Tools/TestWebKitAPI/Configurations/TestWebKitAPIBase.xcconfig

```diff
@@ -26,9 +26,9 @@ GCC_ENABLE_OBJC_EXCEPTIONS = YES;
 
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GTEST_API_=
 
-FRAMEWORK_SEARCH_PATHS = $(FRAMEWORK_SEARCH_PATHS_$(WK_COCOA_TOUCH));
-FRAMEWORK_SEARCH_PATHS_ = $(inherited) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks $(SYSTEM_LIBRARY_DIR)/Frameworks/WebKit.framework/Versions/A/Frameworks;
-FRAMEWORK_SEARCH_PATHS_cocoatouch = $(inherited);
+SYSTEM_FRAMEWORK_SEARCH_PATHS = $(SYSTEM_FRAMEWORK_SEARCH_PATHS_$(WK_COCOA_TOUCH));
+SYSTEM_FRAMEWORK_SEARCH_PATHS_ = $(inherited) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks $(SYSTEM_LIBRARY_DIR)/Frameworks/WebKit.framework/Versions/A/Frameworks;
+SYSTEM_FRAMEWORK_SEARCH_PATHS_cocoatouch = $(inherited);
```

Comment:
We could perhaps have `$(inherited)` in the first `SYSTEM_FRAMEWORK_SEARCH_PATHS` line here, and avoid having to define `SYSTEM_FRAMEWORK_SEARCH_PATHS_cocoatouch` at all.

Also, it's concerning that some projects have `$(inherited)` first, and others last.